src/entities/SecurityToken/Restrictions.ts

```diff
@@ -1,10 +1,179 @@
-import { ModuleName } from '@polymathnetwork/contract-wrappers';
-import { ErrorCode } from '../../types';
-import { ModifyMaxHolderCount } from '../../procedures';
+import {
+  ModuleName,
+  BigNumber,
+  PercentageTransferManagerEvents,
+} from '@polymathnetwork/contract-wrappers';
+import { keys } from 'lodash';
+import {
+  ErrorCode,
+  PercentageWhitelistEntry,
+  ModifyPercentageExemptionsProcedureArgs,
+} from '../../types';
+import {
+  ModifyMaxHolderCount,
+  ModifyMaxHolderPercentage,
+  ModifyPercentageExemptions,
+} from '../../procedures';
 import { SubModule } from './SubModule';
 import { PolymathError } from '../../PolymathError';
+import { TransactionQueue } from '../TransactionQueue';
+
+interface WhitelistParams {
```

Comment:
This should be renamed to something referring to PercentageTM. There are many modules with whitelists.